### PR TITLE
[PPM-285] Remove ACTION_CONTROL_CONTROLLER_PING_REQUEST/RESPONSE

### PR DESCRIPTION
**Pull request by VladyslavTupikin**
Need to remove next messages and all related source code:

```
ACTION_CONTROL_CONTROLLER_PING_REQUEST
ACTION_CONTROL_CONTROLLER_PING_RESPONSE
```

https://jira.prplfoundation.org/browse/PPM-285

### CODE REVIEW

**Comment by simonbaren** [comment]
You need to reorder the commits here.   
Your first commit removes the classes from the YAML files and only later commits remove the usage. It will not compile separately.

**Comment by VladyslavTupikin** [comment]
I have reversed the commits, now I am removing implementation after - messages.

**Comment by morantr** [comment]
This PR should have 3 commits:
1. Remove all the code that use the messages you are going to delete.
2. Remove all messages from yaml files.
3. Update auto-generated code

File: controller/src/beerocks/master/son_master_thread.cpp [hunk 1]
```diff
@@ -2991,69 +2991,6 @@ bool master_thread::handle_cmdu_control_message(const std::string &src_mac,
...
-                }
-                request->total() = response->total();
-                request->seq()   = response->seq() + 1;
-                request->size()  = response->size();
-                if (!request->alloc_data(response->size())) {
-                    LOG(ERROR) << "Failed buffer allocation to size=" << int(response->size());
-                    break;
-                }
-                memset(request->data(), 0, request->data_length());
-                if (!database.update_node_last_ping_sent(hostap_mac)) {
```
Comment (mariomaz) [inline controller/src/beerocks/master/son_master_thread.cpp:2991]: If `update_node_last_ping_sent` method is not used any more, then it should be removed too.
Please check the other database methods that were used in removed code

**Review by mariomaz — APPROVED** [review]
I think some database methods are not used anymore after the code removal

**Review by arnout — APPROVED** [review]
I just *love* a pull request that removes 1000 lines of code :-) Progress!

**Pull request merged**